src/addresses.rs

```diff
@@ -4,7 +4,9 @@ type AddressMap = HashMap<u64, Address>;
 type ChainMap = HashMap<u64, ChainAddresses>;
 type ChainAddress = HashMap<u64, Address>;
 
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, Debug)]
+
+/// Represents the addresses of various core contracts of uniswap on a network.
```

Comment:
```suggestion
/// Represents the addresses of various core contracts of uniswap on a network.
#[derive(Clone, Copy, Debug)]
```
Put derive attributes next to the struct.